### DISCUSSION TITLE
Get Count of Relation instead of Id

### DISCUSSION THREAD

**Discussion by manishonc** [pocketbase/pocketbase#3007] [Q&A]
I was wondering if there is any way I can get how many items the relation has instead of related items ID.

Current 
RelItems [id-1,id2,id3,]

I want 
 RelItems - 3

If it's possible.

**Comment by ganigeorgiev** [comment]
What is the use case for this? Wouldn't a simple client-side count suffice (eg. `relItems.length`)?

There is no builtin functionality to return the rel count instead of the ids and if you want to perform aggregations you can create a helper [View collection](https://pocketbase.io/docs/collections/#view-collection).
